gui/default/syncthing/core/syncthingController.js

```diff
@@ -127,6 +150,23 @@ angular.module('syncthing.core')
             });
         };
 
+        $scope.getLocationHost = function () {
+            return $location.host();
+        };
+
```

Comment:
Leftover? Couldn't find it anywhere.
```suggestion
```

Reply:
Thanks, fixed in 97bf72e67d22839bfeb92c299db0488233b818fc.